docs/core/event_handler/api_gateway.md

```diff
@@ -460,9 +460,22 @@ In the following example, we use a new `Header` OpenAPI type to add [one out of
 
 #### Serializing objects
```

Comment:
```suggestion
#### Supported types for response serialization
```

rename: aligning section with the content.

I've ran the docs locally and this section was confusing me with `Custom serializer`, and whether this was about request serialization. 

I'd suggest changing the name to match the table content and our intent (use case driven).

Reply:
Done